packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -734,7 +764,7 @@ function WorklistDataGrid(props) {
 
   // -------------------------------------
   return (
-    <div className={cx('worklist-data-grid-container')}>
+    <div ref={gridContainerRef} className={cx('worklist-data-grid-container')}>
```

Comment:
Extended to my previous comment: This component's getting really long. Maybe in the future we could look at refactoring a lot of this logic or even splitting up some of the shared responsibilities of this component.

Reply:
@kenk2 - We will be creating the DataGrid component that we have been discussing.  That will break up the logic.